limitador-server/src/http_api/server.rs

```diff
@@ -32,6 +41,10 @@ impl RateLimitData {
     fn metrics(&self) -> &PrometheusMetrics {
         self.metrics.as_ref()
     }
+
+    fn status(&self) -> Status {
+        *self.status.read().unwrap()
```

Comment:
What's your policy on unwraps and possible panics?

Reply:
See [below](https://github.com/Kuadrant/limitador/pull/412#discussion_r1935555048)